test/e2e/specs-playwright/wp-log-in-out-spec.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+
+import playwright from 'playwright';
+
+import LoginPage from '../lib/pages/login-page';
+
+describe( `Auth Screen @canary @parallel`, function () {
+	this.timeout( 30000 );
+	let page;
+	let browser;
+
+	before( 'Start browser', async function () {
+		browser = await playwright.chromium.launch( {
+			args: [ '--disable-dev-shm-usage' ],
```

Comment:
FYI, when playing with WebDriver, I found that removing this flag (our docker image supports SHM) yield to fewer crashes.

Reply:
Right, I have removed this flag from subsequent patches but I think I will remove it here while I'm rebasing.